src/Command/TranskribusAuthCommand.php

```diff
@@ -37,6 +37,7 @@ protected function configure(): void {
 	 * Method that runs the CLI command.
 	 * @param InputInterface $input
 	 * @param OutputInterface $output
+	 * @return int
```

Comment:
The method still returns int

Reply:
I believe `Command::SUCCESS` or `Command::FAILURE` return integers (as mentioned [in the docs](https://symfony.com/doc/current/console.html#creating-a-command))